src/org/zaproxy/zap/view/widgets/WritableFileChooser.java

```diff
@@ -26,6 +28,21 @@ public WritableFileChooser(File currentDirectory) {
 	public void approveSelection() {
 		File selectedFile = getSelectedFile();
 
+        File checkFile = selectedFile;
+        boolean fileExists = checkFile.exists() ? true : false;
```

Comment:
It can just use the return value of method exists().
The condition "selectedFile.exists()" in a following if statement can/should be replaced with this variable.